amgut/handlers/human_survey_completed.py

```diff
@@ -1,8 +1,69 @@
+from logging import Logger
+
 from tornado.web import authenticated
 
 from amgut.lib.util import external_surveys
 from amgut.handlers.base_handlers import BaseHandler
-from amgut import media_locale
+from amgut import media_locale, text_locale
+from amgut.lib.mail import send_email
+from amgut.connections import ag_data
+
+
+def build_consent_form(consent_info):
```

Comment:
My vote is that we make this method public if/when that functionality is added but no strong feelings to be honest